executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model/sections/ProjectViewExcludableListSection.kt

```diff
@@ -16,3 +16,13 @@ data class ProjectViewTargetsSection(
         const val SECTION_NAME = "targets"
     }
 }
+
+data class ProjectViewDirectoriesSection(
+        override val values: List<BuildTargetIdentifier>,
```

Comment:
i think that directories are just `Path` or `String` - it has to support `.` for example